### ISSUE TITLE
map is not working inside Row

### ISSUE THREAD

**Issue by vorlov** [henrybuilt/react-sticky-table#4]
```
<Row>
   <Cell>
      <Checkbox />
    </Cell>
    { headers.map(header =>
      <Cell
        key={ header.key }
      >
        { header.name }
      </Cell>
    )}
</Row>
```
that code throws Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in. Could you please assist?

**Comment by maxhudson** [comment]
Check the answer with 180 upvotes here: http://stackoverflow.com/questions/34130539/uncaught-error-invariant-violation-element-type-is-invalid-expected-a-string

If headers is just an array of objects like this: `[{key: 1, name: "string"}]` there shouldn't be any issue. The issue is likely with your import of the Row/Cell class or setting `key` or `name` to undefined (inadvertantly most likely).

If that doesn't resolve your issue can you please include your entire example or a simplified version of it with the imports and sample of headers?

**Comment by vorlov** [comment]
Hi, problem is not with the import cause if I remove map from render in works, all keys and names in every object in array have values

**Comment by maxhudson** [comment]
@vorlov The issue is related to using a `<Cell></Cell>` as a sibling with an array of `<Cell>`.

The row's children end up being something like `[<Cell />, [<Cell />, <Cell />...]]` which 1.1.8 doesn't support. I'm looking into a permanent solution, but what you should do for now is have an array of cells like so:
```
<Row>
  {cells}
</Row>
```
where cells is an array of this format `[<Cell />, <Cell />]` which you construct above the return of your render method.

**Comment by maxhudson** [comment]
@vorlov this is resolved in v1.1.9 which we just pushed.

**Comment by vorlov** [comment]
@maxhudson thanks a lot! fixed my issue!)
